### PR TITLE
Update Set-VMHost: accurate info on default folders

### PR DESCRIPTION
**Pull request by skycommand**
Areas of change:
- Example 4
    - Fix one typo
    - Give more accurate info about what's stored where
- -VirtualHardDiskPath
    - Explain that it is not a functional setting
- -VirtualMachinePath
    - Explain that it is a functional setting and Hyper-V creates a "Virtual Machines" sub-folder

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/hyper-v/Set-VMHost.md [hunk 1]
```diff
@@ -369,7 +372,7 @@ Accept wildcard characters: False
 ```
 
 ### -VirtualHardDiskPath
-Specifies the default folder to store virtual hard disks on the Hyper-V host.
+Specifies the default folder to store virtual hard disks on the Hyper-V host. This is not a functional setting. The Hyper-V Manager snap-in and Windows Admin Center query it to offer you a default path, but the Hyper-V API (e.g. `CreateFixedVirtualHardDisk`) and `New-VHD` don't use it.
```
Comment (JohanFreelancer9) [inline docset/windows/hyper-v/Set-VMHost.md:375]:
```suggestion
Specifies the default folder to store virtual hard disks on the Hyper-V host. This is not a functional setting. The Hyper-V Manager snap-in and Windows Admin Center query it to offer you a default path, but the Hyper-V API (for example `CreateFixedVirtualHardDisk`) and `New-VHD` don't use it.
```

File: docset/windows/hyper-v/Set-VMHost.md [hunk 2]
```diff
@@ -425,7 +428,7 @@ Accept wildcard characters: False
 ```
 
 ### -VirtualMachinePath
-Specifies the default folder to store virtual machine configuration files on the Hyper-V host.
+Specifies the default folder to store virtual machine configuration files on the Hyper-V host. This is a functional setting, meaning that if you do not supply a path to VM creations tools (e.g. `New-VM`), Hyper-V will use this path. Hyper-V will create at least one subfolder in this path, called "Virtual Machines".
```
Comment (JohanFreelancer9) [inline docset/windows/hyper-v/Set-VMHost.md:431]:
```suggestion
Specifies the default folder to store virtual machine configuration files on the Hyper-V host. This is a functional setting, meaning that if you do not supply a path to VM creation tools (for example `New-VM`), Hyper-V will use this path. Hyper-V will create at least one subfolder in this path, called "Virtual Machines".
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]
Thanks @skycommand!

**Comment by dariomws** [comment]
I think the same information could be added to 2012 and 2012 R2 articles.

**Comment by skycommand** [comment]
@dariomws Done.

**Comment by dariomws** [comment]
Thanks @skycommand!

**Review by tseward — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Pull request merged**